src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -85,4 +85,8 @@ input.usa-only-phone {
   .address-validation-input {
     height: 75%;
   }
+
+  [type=radio] + label::before {
+    min-width: 1.4rem;
```

Comment:
@erikphansen or @ncksllvn, do you know if there's a variable we could use here instead of the `1.4rem`?

Reply:
For context - I needed to add this to make sure the checkbox does not appear stretched when `{street}` is such a long string that it covers more than a single line.

Reply:
Not really, @kelsonic. Looks like doing `units(1.5)` would result in 1.2rem and `units(2)` would result in 1.6rem https://design.va.gov/design/spacing-units